core/services/chainlink/config_general.go

```diff
@@ -386,6 +390,33 @@ func (g *generalConfig) SolanaEnabled() bool {
 	return false
 }
 
+// Currently plugins are only supported on Solana configs
```

Comment:
Needs slight change to return error on misconfiguration even if Solana config specifies to use a plugin - will update shortly

Reply:
Fixed